### ISSUE TITLE
makeflow hangs on timeout when specifying directly -t  in BATCH_OPTIONS

### ISSUE THREAD

**Issue by btovar** [cooperative-computing-lab/cctools#2418]
The issue is that on timeout, slurm is not updating the status file of the job. E.g:

```sh
$ cat slurm.status.6502438
start 1600959872
```
but:

```sh
squeue -u $USER
JOBID   PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
```
and:

```sh
$ sacct
6502438       makeflow0     normal tg-phy130+        272    TIMEOUT      0:0 
```

**Comment by bcdarwin** [comment]
I have this issue as well with `makeflow -Tslurm` using `cctools 7.1.6` and a rather old `slurm 17.02.7'.

**Comment by dthain** [comment]
@btovar recall that the status file is not created by SLURM per se, but by the wrapper script around the job.  So, if SLURM decides on its own to cancel the whole job, then the wrapper script doesn't get a chance to write the completion record.

**Comment by btovar** [comment]
Ah yes, I was treating it as condor log.

**Comment by btovar** [comment]
Fixed with #2419
